src/world.cpp

```diff
@@ -117,12 +117,8 @@ World::World(Json::Value materials_json, Json::Value biome_data, uint32_t x_tile
 
     std::cout << "start of terrain\n";
 
-    constexpr int macro_tile_size = 32;
-    constexpr int height = 128;
-    constexpr int seed = 5;
```

Comment:
You should pass in the seed as a command line argument, and if not specified use a random number

Reply:
passing in a seed is not height on the priority list